common/globals.lua

```diff
@@ -8,6 +8,7 @@ RegionDefs = require('common/region-defs')
 ResourceDefs = require("common/resource-defs")
 BasePath = abyss.getConfig("#Abyss", "BasePath")
 MPQRoot = abyss.getConfig("OpenDiablo2", "MPQRoot")
+IsOnButton = false
```

Comment:
I'm worried about this being a global.. Can we move it to a specific screen?

Reply:
This kinda needs to be global